src/pyscipopt/scip.pxi

```diff
@@ -5533,6 +5556,112 @@ cdef class Model:
         assert isinstance(var, Variable), "The given variable is not a pyvar, but %s" % var.__class__.__name__
         PY_SCIP_CALL(SCIPchgVarBranchPriority(self._scip, var.scip_var, priority))
 
+    def startStrongBranching(self):
+        """Start strong branching. Needs to be called before any strong branching. Must also later end strong branching.
+        If propagation is enabled then strong branching is not done of the LP, but on additionally created nodes (has some overhead)"""
+
+        PY_SCIP_CALL(SCIPstartStrongbranch(self._scip, False))
+
+    def endStrongBranching(self):
+        """End strong branching. Needs to be called if startStrongBranching was called previously.
+        Between these calls the user can access all strong branching functionality. """
+
+        PY_SCIP_CALL(SCIPendStrongbranch(self._scip))
+
+    def getVarStrongBranchInfoLast(self, Variable var):
+        """Get the results of the last strong branching call on this variable (potentially was called
+        at another node).
+
```

Comment:
For consistency with the rest of the codebase's sake, add the ```:param Variable var: ....``` to the docstring. I don't dislike having additional info in the docstring (down, up, etc.), but I wonder if they shouldn't be inline comments

Reply:
Done. As a reminder: We should talk this Friday about the docstrings